.github/CODEOWNERS

```diff
@@ -1,18 +1,16 @@
 # The SVM team is in the process of migrating these subdirectories to a new
 # repo and would like to avoid introducing dependencies in the meantime.
-/builtins-default-costs/ @anza-xyz/svm
-/compute-budget/ @anza-xyz/svm
 /compute-budget-instruction/ @anza-xyz/fees
 /fee/ @anza-xyz/fees
 /log-collector/ @anza-xyz/svm
 /program-runtime/ @anza-xyz/svm
 /programs/bpf_loader/ @anza-xyz/svm
-/programs/compute-budget/ @anza-xyz/svm
```

Comment:
there are still some svm-internal conditional cfg in this crate (and builtins-default-costs). Should these be removed before we remove the `svm` team's ownership?

Reply:
`SVM`'s crates do not seem to be calling into these for non-dev code anymore. We can remove the `svm-internal` conditional configs from these crates.